src/components/DeleteModal/DeleteModal.jsx

```diff
@@ -0,0 +1,25 @@
+import useStore from '../../hooks/useStore';
+import StyledButton from '../StyledButton';
+import StyledSection from '../StyledSection';
+
+export default function DeleteModal({ onDelete, videoId }) {
+	const deleteVideo = useStore(state => state.deleteVideo);
+
+	return (
+		<StyledSection>
+			<h2>Are you sure you want to delete this video?</h2>
+			<StyledButton type="button" onClick={onDelete}>
```

Comment:
The name `onDelete` is a bit confusing if this is the function which cancels and closes the modal. I would suggest to name it `onCancel`:
```suggestion
export default function DeleteModal({ onCancel, videoId }) {
	const deleteVideo = useStore(state => state.deleteVideo);

	return (
		<StyledSection>
			<h2>Are you sure you want to delete this video?</h2>
			<StyledButton type="button" onClick={onCancel}>
```

Reply:
Done, thanks!